setup.py

```diff
@@ -199,6 +212,8 @@ def get_build_version():
         include_package_data=True,
         package_data=package_data,
         install_requires=install_requires,
+        ext_modules=ext_modules,  # for fp8
```

Comment:
This looks to be an FP8-only extension. How about using an environment variable(like `USE_FP8 = 1`, default is 0) to decide whether or not to build it? This way won't impose additional overhead on users who don't require FP8 functionality.

Reply:
good point